docroot/resources/JSON/celestial-bodies/soho/jupiter/soho_jupiter_2000-04-25_2000-05-22.json

```diff
@@ -0,0 +1 @@
ance_observer_to_body_au": 5.973807173173043, "distance_sun_to_observer_au": 0.9935962341879991, "distance_sun_to_body_au": 4.988169861597884, "behind_plane_of_sun": "True", "x": 23856.590833722435, "y": -421.9086185024087}, "958511980945": {"distance_observer_to_body_au": 5.973780637722771, "distance_sun_to_observer_au": 0.9936022131991185, "distance_sun_to_body_au": 4.988174063721821, "behind_plane_of_sun": "True", "x": 23911.704322462367, "y": -414.8739897899437}, "958513780932": {"distance_observer_to_body_au": 5.973754016320653, "distance_sun_to_observer_au": 0.9936081917135763, "distance_sun_to_body_au": 4.988178266057242, "behind_plane_of_sun": "True", "x": 23966.817172197738, "y": -407.8383550577857}, "958515580919": {"distance_observer_to_body_au": 5.973727309829642, "distance_sun_to_observer_au": 0.993614170467398, "distance_sun_to_body_au": 4.98818246860413, "behind_plane_of_sun": "True", "x": 24021.9291897377, "y": -400.80174105489385}, "958517380905": {"distance_observer_to_body_au": 5.973700518315924, "distance_sun_to_observer_au": 0.9936201496253766, "distance_sun_to_body_au": 4.988186671362491, "behind_plane_of_sun": "True", "x": 24077.04052845422, "y": -393.7640963056036}, "958519180892": {"distance_observer_to_body_au": 5.973673641763257, "distance_sun_to_observer_au": 0.993626129164848, "distance_sun_to_body_au": 4.988190874332313, "behind_plane_of_sun": "True", "x": 24132.151186262403, "y": -386.7254136590273}, "958520980878": {"distance_observer_to_body_au": 5.9736466796767855, "distance_sun_to_observer_au": 0.9936321084511247, "distance_sun_to_body_au": 4.988195077513595, "behind_plane_of_sun": "True", "x": 24187.260970058414, "y": -379.6858234364993}, "958522780865": {"distance_observer_to_body_au": 5.973619632557536, "distance_sun_to_observer_au": 0.9936380881171725, "distance_sun_to_body_au": 4.988199280906329, "behind_plane_of_sun": "True", "x": 24242.370076277053, "y": -372.64520125290164}, "958524580851": {"distance_observer_to_body_au": 5.973592500364647, "distance_sun_to_observer_au": 0.9936440680173972, "distance_sun_to_body_au": 4.988203484510518, "behind_plane_of_sun": "True", "x": 24297.47835856895, "y": -365.60361465252777}, "958526380838": {"distance_observer_to_body_au": 5.9735652822418945, "distance_sun_to_observer_au": 0.993650047416593, "distance_sun_to_body_au": 4.988207688326153, "behind_plane_of_sun": "True", "x": 24352.586013877353, "y": -358.5610427421659}, "958528180824": {"distance_observer_to_body_au": 5.9735379795447745, "distance_sun_to_observer_au": 0.993656027582738, "distance_sun_to_body_au": 4.988211892353224, "behind_plane_of_sun": "True", "x": 24407.692902635546, "y": -351.5174698351151}, "958529980810": {"distance_observer_to_body_au": 5.973510591783066, "distance_sun_to_observer_au": 0.9936620079798624, "distance_sun_to_body_au": 4.988216096591732, "behind_plane_of_sun": "True", "x": 24462.79897245809, "y": -344.47294142292856}, "958531780797": {"distance_observer_to_body_au": 5.9734831181
```

Comment:
Are these resources files required? I don't see any references to "celestial-bodies" but I do see references to "celestial-objects" and it is in the .gitignore.

Reply:
I added these to the gitignore. If we need them later they can be added back. But for now I think this is ok.